src/utils/store.ts

```diff
@@ -0,0 +1,120 @@
+import { Graph } from 'ngraph.graph';
```

Comment:
don't love this file name as it couples this logic that is somewhat generic to the store 'context' but its fine for now.